### DISCUSSION TITLE
Include "Onboarding" project in selfhosted penpot

### DISCUSSION THREAD

**Discussion by miklobit** [uxbox/uxbox#612] [Q&A] [answered]
Is any way to import "Onboarding" project into selfhosted installation ? 
Or include this project in docker image ? 
![Screenshot(50)](https://user-images.githubusercontent.com/1035161/107362140-ac3b9c00-6ad8-11eb-9b50-26c081a13474.png)

**Comment by myfunnyandy** [comment, marked as answer]
I'm afraid that is not possible, for the moment those files are only available at the cloud version.

**Reply by niwinz** [reply]
They will be posible to have in the selfhosted environment when we implement the export (a complete file export)
